src/utils/graphql/queries/getReviewInfo.query.ts

```diff
@@ -0,0 +1,32 @@
+import { gql } from '@apollo/client'
+
+export default gql`
+  query getReviewInfo($reviewerId: Int!, $applicationId: Int) {
+    reviewAssignments(
+      condition: { reviewerId: $reviewerId, applicationId: $applicationId }
+      orderBy: TIME_CREATED_DESC
+    ) {
+      nodes {
+        id
+        status
+        timeCreated
+        reviews {
+          nodes {
+            id
+            level
```

Comment:
i think it's better to get level from reviewAssignment

Reply:
Yea, good point

Reply:
532de2d